src/constants/routes.ts

```diff
@@ -1,3 +1,4 @@
-export const MAIN_PAGE = '/';
+export const HOME_PAGE = '/';
```

Comment:
Please create task to rename `Home` module to the `Project` module. `Home` is too abstract name and `Project` is more eligible. And will start it after merging this task to master to avoid a lot of conflicts